src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -179,9 +179,20 @@ static void _applyComposition(Paint* paint, const SvgNode* node, float vx, float
 {
     if (node->style->comp.method == CompositeMethod::None) return;
 
+    /* Do not drop in Circular Dependency.
+       Composition can be applied recursively if its children nodes have composition target to this one. */
+    if (node->style->comp.applying) {
```

Comment:
Missing initialization for this member.

Reply:
@JSUYA  here? node->style = (SvgStyleProperty*)calloc(1, sizeof(SvgStyleProperty))

Reply:
ah i forgot it.